packages/wrangler/src/dev/start-server.ts

```diff
@@ -148,7 +172,34 @@ export async function startDevServer(
 		),
 	});
 
+	console.log({ experimentalDevenvRuntime: props.experimentalDevenvRuntime });
+
+	if (props.experimentalDevenvRuntime) {
```

Comment:
what about the logs in this `if` block? are these meant to be kept as well?

Reply:
No, I don't think any are—I'll go through and remove